### PR TITLE
adding saveAs pattern

### PR DESCRIPTION
**Pull request by Bioninbo**

### CODE REVIEW

File: publish-process-outputs-subdirectories/main.nf [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+ /*
+  * author Paolo Di Tommaso <paolo.ditommaso@gmail.com>
```
Comment (pditommaso) [inline publish-process-outputs-subdirectories/main.nf]: Thank a lot for the contribution. There's should be your name here, you wrote that ;)

File: publish-process-outputs-subdirectories/main.nf [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  < ${samples[0]} zcat > sample1.fq
+  < ${samples[1]} zcat > sample2.fq
+
+  awk '{s++}END{print s/4}' sample1.fq > sample1_readCounts.txt
+  awk '{s++}END{print s/4}' sample2.fq > sample2_readCounts.txt
+
+  head -n 50 sample1.fq > sample1_outlook.txt
+  head -n 50 sample2.fq > sample2_outlook.txt
+
+  """
```
Comment (pditommaso) [inline publish-process-outputs-subdirectories/main.nf]: Please ident better the code snippet (have a look the the other examples for reference).

**Comment by pditommaso** [comment]
Tests are failing. Have tried to run this snippet?

File: publish-process-outputs-subdirectories/main.nf (hunk 1 above)
Comment (Bioninbo) [reply publish-process-outputs-subdirectories/main.nf]: Don't worry, it's really not a major contribution... :)

**Comment by pditommaso** [comment]
Finally I've decided to merge your example with #9. The ability of using multiple `pubishDir`, introduced recently, I think should be preferred over the use of `saveAs`. Thanks for your contribution!

**Pull request merged**